src/components/atoms/favourite-tile.astro

```diff
@@ -0,0 +1,78 @@
+---
+import { Image } from '@astrojs/image/components';
+
+import Typography from '@/atoms/typography.astro';
+import type FavouriteElement from '@/types/favourites-section';
+
+type FavouriteElementVariant = 'book' | 'media' | 'video' | 'person';
+
+const variantToTileWidth = {
+  book: 'w-51',
+  media: 'w-29',
+  video: 'w-70',
+  person: 'w-29',
+};
+
+const variantToWidth = {
+  book: 200,
+  media: 122.66,
+  video: 277.33,
+  person: 122.66,
+};
+
+const variantToHeight = {
+  book: 260,
+  media: 122.66,
+  video: 156,
+  person: 122.66,
+};
+
+export interface Props extends astroHTML.JSX.HTMLAttributes {
+  value: FavouriteElement;
+  variant: FavouriteElementVariant;
+}
+
+const { value, variant, ...props } = Astro.props;
+const FavouriteTile = 'a';
+---
+
+<FavouriteTile
+  href={value.url}
+  class:list={[
+    variant,
+    props.class,
+    'flex',
+    'group',
+    'flex-col',
+    'gap-3',
+    'transition',
+    'duration-300',
+    'hover:translate-y-2',
+    variantToTileWidth[variant],
+  ]}
+>
+  <Image
+    class:list={[
+      'rounded-lg',
+      'shadow-md',
+      'object-cover',
+      variant === 'person' && 'grayscale',
+      'hover:grayscale-0',
+      'transition',
+      'duration-300',
+    ]}
+    src={value.image || value.cover || `https://img.youtube.com/vi/${value.url.split('/').pop()}/0.jpg`}
```

Comment:
To liczenie miniaturki wyciągnąłbym do jakieś funkcji w TS, łatwiej by się czytało coś takiego:
```suggestion
    src={value.image || value.cover ||  getYoutubeThumbnail(value.url)}
```

Reply:
Done ;-)